tests/test_app/polls/models.py

```diff
@@ -29,7 +29,15 @@ class Choice(models.Model):
     """
 
     poll = models.ForeignKey(Poll, on_delete=models.CASCADE)
-    choice_text = models.CharField(max_length=255)
+    choice_text = models.CharField(
+        max_length=255,
+        choices=(
+            ("Heres a choice", "Heres a choice"),
+            ("Heres another choice", "Heres another choice"),
```

Comment:
Typos `Here is` 

As it is visible to everyone, I would change them

Reply:
hahaha, forever my personal spelling and grammar checker